j5/base_robot.py

```diff
@@ -0,0 +1,20 @@
+"""A base class for robots."""
+
+import atexit
+
+from j5.boards import Board
+
+
+class BaseRobot:
+    """A base robot."""
+
+    def make_self(self):
+        """Make this robot safe."""
+        BaseRobot.make_all_safe()
+
+    @staticmethod
+    @atexit.register
+    def make_all_safe():
```

Comment:
Seeing as this method is static, would it make more sense to move this method to the `Board` class?

Reply:
I agree.